src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs

```diff
@@ -30,7 +30,7 @@ public class KestrelServerOptions
 
         /// <summary>
         /// Enables the Listen options callback to resolve and use services registered by the application during startup.
-        /// Typically initialized by <see cref="Hosting.WebHostBuilderKestrelExtensions.UseKestrel(Hosting.IWebHostBuilder, Action{KestrelServerOptions})"/>.
+        /// Typically initialized by UseKestrel()"/>.
```

Comment:
You wan't to put KestrelServerOptions in the meta package or Core couldn't read those options. The UseKestrel() extension method needs to be in the meta package so it can reference both Core and configure the default libuv transport.

@davidfowl Do you think we should have a WebHostBuilder extension method for using Kestrel without wiring the default libuv transport? What should it be called?